projects/demo/src/app/app.component.html

```diff
@@ -66,7 +66,7 @@ <h2 id="demos">Demos</h2>
       heading="Responsive - resize me!"
       stackblitz="https://stackblitz.com/edit/ngx-doe-gallery-demo-responsive?file=src%2Fapp%2Fapp.component.html"
     >
-      <doe-gallery [items]="images" loading="lazy"></doe-gallery>
+      <doe-gallery [items]="images" [isRtl]="true" loading="lazy"></doe-gallery>
```

Comment:
I will create a separate demo example with the RTL on later, so you can remove it here.